ouroboros-consensus-test/test-consensus/Test/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -435,6 +466,14 @@ runChainSync securityParam (ClientUpdates clientUpdates)
       tick <- atomically $ LogicalClock.getCurrentTick clock
       traceWith tr (tick, ev)
 
+    catchChainSyncClient :: m a -> (ChainSyncClientException -> m a) -> m a
+    catchChainSyncClient ma handler = ma `catch` \case
```

Comment:
Minor: maybe `catches` would be more idiomatic/recognizable? It'd save one layer of `fromException`, but not both layers.

Reply:
Agreed, I took the opportunity to slightly simplify and generalize this function :+1: